includes/wp-cli/class-cache.php

```diff
@@ -1,32 +0,0 @@
-<?php
-/**
- * Cache commands for WP-CLI
- *
- * @package a8c_Cron_Control
- */
-
-namespace Automattic\WP\Cron_Control\CLI;
-
-/**
- * Manage Cron Control's internal caches
- */
-class Cache extends \WP_CLI_Command {
-	/**
-	 * Flush the cache
-	 *
-	 * @subcommand flush
-	 * @param array $args Array of positional arguments.
-	 * @param array $assoc_args Array of flags.
-	 */
-	public function flush_internal_caches( $args, $assoc_args ) {
-		$flushed = \Automattic\WP\Cron_Control\_flush_internal_caches();
```

Comment:
I want to stop exposing this detail from the store, as it can be handled solely from within the store anytime DB interactions take place.

At first was going to do a `WP_CLI::error( "deprecated msg..." )`, but realized you get basically the same result by removing the CLI entirely. WP just shows like "Error: 'X' is not a registered wp command.", which feels more than sufficient.

Removing in general also feels a lot safer in CLI context.